src/Application/Renderer/SliceShader.cc

```diff
@@ -77,6 +77,7 @@ bool SliceShader::post_initialize()
   this->opacity_loc_ = this->get_uniform_location( "opacity" );
   this->mask_mode_loc_ = this->get_uniform_location( "mask_mode" );
   this->scale_bias_loc_ = this->get_uniform_location( "scale_bias" );
+  this->pick_color_loc_ = this->get_uniform_location("pick_color");
```

Comment:
Is `post_initialize` always called after constructing a `SliceShader`? Either way, the `SliceShader` constructor should initialize all of its integer variables to 0.

Reply:
here https://github.com/SCIInstitute/Seg3D/pull/278/files#diff-b0f30d9b11ff1d4956aa4e163e0b730dL39

Reply:
Changed